src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyCommentBeforeLeftCurly2.java

```diff
@@ -25,29 +30,29 @@ public class InputLeftCurlyCommentBeforeLeftCurly2 {
     private long[] countList;
 
     void method1() /* comment
-    comment */ { // ok until #11410
+    comment */ { // violation ''{' at column 16 should be on the previous line'
```

Comment:
message doesnot make sense now , as we can not put '{' at EOL as there is multiline comment at the end.

Reply:
ok i will change it to give no violation in this case

Reply:
it should be no violation as '{' is at the end-of-line  (considering note about comments that are not considered as text ).

and this become interesting, that we do consider comments as text sometime and sometime not. It is very strange.

Reply:
@romani what to do in this case 
```
void method()  /* comment */ 
/* comment */ {
}
```

what to consider violation or not ?

Reply:
done 

currently it is showing violation